pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -3174,3 +3180,60 @@ def test_last_week_of_month_on_offset():
     slow = (ts + offset) - offset == ts
     fast = offset.onOffset(ts)
     assert fast == slow
+
+
+@pytest.mark.parametrize('box, assert_func', [
```

Comment:
can you use ``tm.assert_equal`` instead

Reply:
I am testing 2 array like and 1 scalar here, and `tm.assert_equal` is not defined for scalars:

`https://github.com/pandas-dev/pandas/blob/master/pandas/util/testing.py#L1497`